avalanche-ops/src/aws/cfn-templates/asg_ubuntu.yaml

```
@@ -201,6 +201,12 @@ Parameters:
     MaxValue: 100
     Description: 0 for Spot only. 100 for On-Demand only.
 
+  SSHEnabled:
```

```suggestion
  SshEnabled:
```
to be consistent with others? :)